migration/mig_0013_add_rule_hit_table.go

```diff
@@ -30,11 +27,55 @@ var mig0013AddRuleHitTable = Migration{
 		CREATE TABLE rule_hit (
 			org_id          INTEGER NOT NULL,
 			cluster_id      VARCHAR NOT NULL,
-			rule_id         VARCHAR NOT NULL,
-			error_key       VARCHAR NOT NULL,
-			report          VARCHAR NOT NULL,
-			PRIMARY KEY(cluster_id, org_id, rule_id, error_key)
+			rule_fqdn       VARCHAR NOT NULL,
+			rule_key       	VARCHAR NOT NULL,
```

Comment:
Why this was renamed to `rule_key`?

Reply:
@tisnik, what do you think? Should it be `error_key` or `rule_key`? Can't be just `key` because it's a reserved keyword in sql.

Reply:
What's it called in the OCP Rules Engine documentation? :thinking:

Reply:
🤔

Reply:
According to what I see in our OCP documentation. There is some ERROR_KEY, but it's not very clear what is it. https://gitlab.cee.redhat.com/ccx/ccx-rules-ocp/-/blob/master/docs/development_guidelines.md

Reply:
Looks like it's better to postpone the decision. Reverted it back